src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/solution/monitoring_step.py

```diff
@@ -164,8 +163,8 @@ class MonitoringStep(StepModel):
     @transaction(
         problem_setup_step=StepSpec(
             download=[
-                "tcp_server_host",
-                "tcp_server_port",
+                "osl_server_host",
```

Comment:
Do you really need the host and port info?

Reply:
Yes.